src/plugins/vis_type_timelion/server/series_functions/es/index.js

```diff
@@ -96,20 +96,16 @@ export default new Datasource('es', {
       kibana: true,
       fit: 'nearest',
     });
-
-    const findResp = await tlConfig.savedObjectsClient.find({
-      type: 'index-pattern',
-      fields: ['title', 'fields'],
-      search: `"${config.index}"`,
-      search_fields: ['title'],
-    });
-    const indexPatternSavedObject = findResp.saved_objects.find((savedObject) => {
-      return savedObject.attributes.title === config.index;
+    const indexPatternsService = await tlConfig.getIndexPatternsService();
+    const searchIndexPatterns = await indexPatternsService.find(config.index);
```

Comment:
it's not working as expected.  To reproduce that, let's create the following index: 
![image](https://user-images.githubusercontent.com/20072247/106272302-e50a8400-6241-11eb-89f2-d6ed25b85f67.png)

and then let's try to find that index using` indexPatternsService.find` method. I used the following snippet:
```
console.log('f1', await indexPatterns.find('f1'));
console.log('index-a,index-b,index-c,index-d', await indexPatterns.find('index-a,index-b,index-c,index-d'));
```

Result of that execution looks like: 
![image](https://user-images.githubusercontent.com/20072247/106273301-79291b00-6243-11eb-9b8e-edb7c17732ec.png)

it means that in that method we are looking by `title` not by `patternList`